frontend/class-frontend.php

```diff
@@ -1039,6 +1039,15 @@ public function adjacent_rel_links() {
 			return;
 		}
 
+		/**
+		 * Filter: 'wpseo_pre_adjacent_rel_links' - Allows disabling of Yoast adjacent links if this is being handled by other code.
+		 *
+		 * @api bool $links_generated Indicates if other code has handled adjacent links.
+		 */
+		if ( apply_filters( 'wpseo_pre_adjacent_rel_links', false ) ) {
```

Comment:
Could you rename this filter to `wpseo_disable_adjacent_rel_links`, which more accurately describes what you can do with the filter and what return value is expected.

_note: friendly reminder to also update the filter name in the comments above_